test/dump/debug-names.txt

```diff
@@ -87,6 +87,29 @@
 000002a: 36                                        ; FIXUP section size
 debug-names.wasm:	file format wasm 0x00000d
 
+Section Details:
+TYPE:
+ - [0] (i32) -> nil
+ - [1] (f32) -> nil
+FUNCTION:
+ - [0] sig=0
+ - [1] sig=1
+CODE:
+ - func 0
+ - func 1
+USER:
+ - name: "name"
+ - [0] $F1
```

Comment:
It's not immediately clear what these indexes are. Maybe add "func" and "local" somewhere?